### PR TITLE
Split PanZoom stories into individual files

### PR DESCRIPTION
**Pull request by loichuder**
Follow-up of #955

### CODE REVIEW

File: apps/storybook/src/VisCanvasXZoom.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+export const ModifierKey = Template.bind({});
+ModifierKey.args = {
+  disabled: false,
+  modifierKey: 'Shift',
+};
+
+export default {
+  ...VisCanvasStoriesConfig,
+  title: 'Building Blocks/VisCanvas/XZoom',
```
Comment (axelboc) [inline apps/storybook/src/VisCanvasXZoom.stories.tsx:46]: I'd be tempted to remove the nesting under `VisCanvas`, like for `HeatmapMesh`.

**Review by axelboc — APPROVED** [review]
Awesome!!

File: apps/storybook/src/VisCanvasXZoom.stories.tsx (hunk 1 above)
Comment (loichuder) [reply apps/storybook/src/VisCanvasXZoom.stories.tsx:46]: Or a new section `PanZoom` ?

Comment (loichuder) [reply apps/storybook/src/VisCanvasXZoom.stories.tsx:46]: I meant to replace the nesting `VisCanvas` section by a new section `PanZoom`

Comment (axelboc) [reply apps/storybook/src/VisCanvasXZoom.stories.tsx:46]: Yep, sounds good!

**Pull request merged**